content/index.html

```diff
@@ -25,7 +25,7 @@ <h5 class="content-center"><strong>HTTP is inefficient and expensive</strong></h
                         <img src="/images/ipfs-illustration-history.svg" />
                     </div>
                     <h5 class="content-center"><strong>Humanity's history is deleted daily</strong></h5>
-                    <p><a href="https://blogs.loc.gov/digitalpreservation/2011/11/the-average-lifespan-of-a-webpage/">The average lifespan of a web page is 100 days.</a> Remember GeoCities? The web doesn't anymore. It's not good enough for the primary medium
+                    <p><a href="http://blogs.loc.gov/thesignal/2011/11/the-average-lifespan-of-a-webpage/">The average lifespan of a web page is 100 days.</a> Remember GeoCities? The web doesn't anymore. It's not good enough for the primary medium
```

Comment:
Switch back to `https` (mixed content but still works).

Reply:
Perfect. I made this change. Thanks!